server/scratchy_server/filters/mongoexception.py

```diff
@@ -0,0 +1,21 @@
+from flask import abort
+from functools import wraps
+from mongoengine.errors import ValidationError
+import logging
+
+def validation(func): # not entirely satisfied by the name, hoping someone will find a better one
```

Comment:
Rename `func` with `ressource_method`.
Add pydoc to explain it should be use to wrappe a MethodView method.